proto/ignition/msgs/world_stats.proto

```diff
@@ -61,4 +61,7 @@ message WorldStatistics
 
   /// \brief Iteration step size. It's zero when paused.
   Time step_size = 10;
+
+  /// \brief True is the simulator is stepping, as opposed to running.
```

Comment:
so stepping means that it is executing a fixed number of iterations, as opposed to running for an unbounded number of iterations?

Reply:
Yeah - the idea is that this field can be used in combination with the `paused` field of this message to provide more context. If `paused` is `false`, then the process is running/executing, but it's not clear if it's running/executing for a bounded or unbounded number of iterations. Adding a `stepping` field would provide this missing information.

I think that this can be made a little clearer. Perhaps we can provide the context that I just gave above and also add the note I suggested in https://github.com/ignitionrobotics/ign-msgs/pull/199#discussion_r743039123.

Reply:
Yeah, I can add that clarification note.

Reply:
432cb99